### PR TITLE
The string representation of a UTC time includes UTC in it

### PR DESCRIPTION
**Pull request by valeriat**
This pull request fixes issue#44.

### CODE REVIEW

**Comment by vaibhavmagarwal** [comment]
@valeriat : your patch seems to be not rebased on the latest code. Can you please rebase it and send in the pull request. Thanks

File: mozci/utils/tzone.py [hunk 1]
```diff
@@ -62,7 +62,16 @@ def __init__(self):
         USTimeZone.__init__(self, -8, "Pacific",  "PST", "PDT")
 
 
+class Universal(USTimeZone):
```
Comment (armenzg) [inline mozci/utils/tzone.py:65]: Can you please inherit from datetime.tzinfo?

**Comment by armenzg** [comment]
I'm glad to see you made progress in here. Thank you!
A `git pull` is needed as @vaibhavmagarwal pointed out.

File: mozci/utils/tzone.py (hunk 1 above)
Comment (armenzg) [inline mozci/utils/tzone.py:65]: Perhaps USTimeZone should be transformed into a more generic class. I don't know.
Perhaps there are some libraries out there from which we can copy.
Or drop the use of tzone.py and include one of those libraries.
I don't want to scope creep.

**Comment by vaibhavmagarwal** [comment]
Hey @valeriat, let us know if you face any problems dealing with this, and would like our help on this.
You would need to do:
git checkout -b UTC-problem
git rebase master

It will possibly have conflicts that will needed to be solved manually. We normally are present in #ateam on irc.mozilla.org, so drop by if you have questions!

**Comment by valeriat** [comment]
Thank you! I made new pull request.
